docs/static/processing-info.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[processing]]
+=== Processing Details
+
+Understanding how {ls} works and how components interrelate can help you make
+better decisions when you are setting up or adjusting your {ls} environment.
+This section is designed to elevate concepts to assist with that level of
+knowledge.
+
+NOTE: This is a new section. We're still working on it.
+
+[float] 
+[[event-ordering]] 
+==== Event ordering 
+
+By design and by default, {ls} does not guarantee event order. Reordering can
+occur in two places:
+
+* Events within a batch can be reordered during filter processing.
+* In-flight batches can be reordered when one or more batches are processed faster than
+others. 
+
+When maintaining event order is important, use a single worker. 
+This approach ensures that batches are computed one-after-the-other, and
+that events maintain their order within the batch.
```

Comment:
I think this might be slightly misleading to think that "When maintaining event order is important, use a single worker" and using a single worker is all you need. 

Maybe something like:

"When maintaining event order is important, use a single worker and set 'pipeline.ordered => true'.

I think this will allow the this doc to live longer if we ever change the "auto" setting behaviour... WDYT?

Reply:
Nice clarification! Thank you for suggesting it.